src/source/generator.scala

```diff
@@ -361,11 +361,10 @@ abstract class Generator(spec: Spec)
     ns match {
       case "" => f(w)
       case s =>
-        val parts = s.split("::")
-        w.wl(parts.map("namespace "+_+" {").mkString(" ")).wl
+        w.wl(s"namespace $s {").wl
```

Comment:
the only change

Reply:
Thanks!  I agree it's reasonable to generate C++17 at this point.